compiler/rustc_metadata/src/rmeta/decoder/cstore_impl.rs

```diff
@@ -413,10 +413,13 @@ impl CStore {
         let span = data.get_span(id.index, sess);
 
         // Mark the attrs as used
-        let attrs = data.get_item_attrs(id.index, sess);
-        for attr in attrs.iter() {
-            sess.mark_attr_used(attr);
-        }
+        let attrs: Vec<_> = data
+            .get_item_attrs(id.index, sess)
+            .map(|attr| {
+                sess.mark_attr_used(&attr);
```

Comment:
I have some good news - `mark_attr_used` is not actually needed here and can be removed.